utils_cv/tracking/model.py

```diff
@@ -251,33 +324,16 @@ def evaluate(
         Returns:
             strsummary: str output by method in 'motmetrics' package, containing metrics scores        
         """
-
+        
         # Implementation inspired from code found here: https://github.com/ifzhang/FairMOT/blob/master/src/track.py
-        evaluator = Evaluator(gt_root_path, "single_vid", "mot")
-
-        with tempfile.TemporaryDirectory() as tmpdir1:
-            os.makedirs(osp.join(tmpdir1, "results"))
-            result_filename = osp.join(tmpdir1, "results", "results.txt")
-
-            # Save results im MOT format for evaluation
-            bboxes_mot = boxes_to_mot(results)
-            np.savetxt(result_filename, bboxes_mot, delimiter=",", fmt="%s")
-
-            # Run evaluation using pymotmetrics package
-            accs = [evaluator.eval_file(result_filename)]
-
-        # get summary
-        metrics = mm.metrics.motchallenge_metrics
-        mh = mm.metrics.create()
-
-        summary = Evaluator.get_summary(accs, ("single_vid",), metrics)
-        strsummary = mm.io.render_summary(
-            summary,
-            formatters=mh.formatters,
-            namemap=mm.io.motchallenge_metric_names,
-        )
+        accs = []
```

Comment:
[minor] instead of accs=[] and accs.append(mot_accumulator) could just do:
   self.mot_summary([mot_accumulator], ["single_vid"])
The variable name "accs" is also misleading so best to just remove it if possible.